src/main/java/frc/robot/commands/test/DriveFeedForwardCharacterzation.java

```diff
@@ -0,0 +1,87 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands.test;
+
+
+
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.Drivetrain;
+import frc.robot.util.FeedForwardCharacterizationData;
+
+
+/** Add your docs here. */
+public class DriveFeedForwardCharacterzation extends CommandBase {
+  double value = 0;
```

Comment:
include m_, also a better name than "value" would be more clear, probably some name like m_volts, i was confused initially what this was